### PR TITLE
[Nano] new tensorflow quantize api

### PR DESCRIPTION
**Pull request by zhentaocc**
Still WIP.  
Make tensorflow api consistent with pytorch.

### CODE REVIEW

File: python/nano/src/bigdl/nano/tf/quantization.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 
 def quantize(self,
+             precision='int8',
```
Comment (jason-dai) [inline python/nano/src/bigdl/nano/tf/quantization.py:24]: will it be possible to support other accelerators in future (such as openvino)?

Comment (zhentaocc) [reply python/nano/src/bigdl/nano/tf/quantization.py:24]: Yes, sure for openvino. As for onnx, maybe we can follow https://onnxruntime.ai/docs/tutorials/tf-get-started.html. But before that, we should figure out how to wrap a quantized tf model.

Comment (jason-dai) [reply python/nano/src/bigdl/nano/tf/quantization.py:24]: The we may need an `accelerator` parameter?

**Comment by zhentaocc** [comment]
http://10.112.231.51:18888/job/BigDL-PRNV-chronos-Python-Spark-2.4-py37-Horovod/1085/  
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/207/

**Pull request merged**